### PR TITLE
Fixes various gen 3 event templates.

### PR DESCRIPTION
**Pull request by kamronbatman**
Fixes various gen 3 event templates.

### CODE REVIEW

**Comment by kamronbatman** [comment]
@kwsch - Tested and working.  Ready to squash/merge.

**Comment by kamronbatman** [comment]
After I made the changes, I tested again.  Still working.

File: PKHeX.Core/Legality/Tables3.cs [hunk 1]
```diff
@@ -272,37 +272,39 @@ public static partial class Legal
...
             
             // Berry Glitch Fix
             // PCJP - (December 29, 2003 to March 31, 2004)
-            new WC3 { Species = 263, Level = 5, Version = (int)GameVersion.R, Language = 1, Method = PIDType.BACD_R_S, TID = 21121, OT_Name = "ルビー", OT_Gender = 1 },
+            new WC3 { Species = 263, Level = 5, Version = (int)GameVersion.S, Language = 1, Method = PIDType.BACD_R_S, TID = 21121, OT_Name = "ルビー", OT_Gender = 1 },
             new WC3 { Species = 263, Level = 5, Version = (int)GameVersion.S, Language = 1, Method = PIDType.BACD_R_S, TID = 21121, OT_Name = "サファイア", OT_Gender = 0 },
+            
             // EBGames/GameStop (March 1, 2004 to April 22, 2007), also via multi-game discs
-            new WC3 { Species = 263, Level = 5, Version = (int)GameVersion.R, Language = 2, Method = PIDType.BACD_R_S, TID = 30317, OT_Name = "RUBY", OT_Gender = 1 },
+            new WC3 { Species = 263, Level = 5, Version = (int)GameVersion.S, Language = 2, Method = PIDType.BACD_R_S, TID = 30317, OT_Name = "RUBY", OT_Gender = 1 },
```
Comment (javierhimura) [inline PKHeX.Core/Legality/Tables3.cs]: Shouldnt be the ruby shiny zizzagoon events only for ruby games instead of shapphire?

**Comment by javierhimura** [comment]
You have changed the shiny zizagoon event from ruby to saphire? This will make illegal the event received in ruby games.

**Comment by kamronbatman** [comment]
That is a misconception. All berry glitch zigzagoons have an origin game of sapphire.

File: PKHeX.Core/Legality/Tables3.cs (hunk 1 above)
Comment (kamronbatman) [inline PKHeX.Core/Legality/Tables3.cs]: That is a misconception.  I have virtually all of them here and you will see the pokemon with the OTN "RUBY" that are female, have an origin game of Sapphire.
https://github.com/projectpokemon/EventsGallery/tree/master/Gen%203/ENG/Berry%20Glitch%20Shiny%20Zigzagoon

**Pull request merged**